swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
+from .item import Item
+
+
+class Clothing(Item):
+    def __init__(self, condition=0, age=0):
+        super().__init__("Clothing", condition, age)
```

Comment:
👍 good idea hardcoding "clothing" here since that will never change if you are creating an instance of Clothing!